.gitmodules

```diff
@@ -3,13 +3,9 @@
 	url = https://github.com/microsoft/MixedRealityToolkit-Unity
 	branch = mrtk_development
 	ignore = dirty
-[submodule "external/ARCore-Unity-SDK"]
-	path = external/ARCore-Unity-SDK
-	url = https://github.com/google-ar/arcore-unity-sdk
-	ignore = dirty
 [submodule "external/Azure-Spatial-Anchors-Samples"]
 	path = external/Azure-Spatial-Anchors-Samples
-	url = https://github.com/Azure/azure-spatial-anchors-samples
+	url = https://github.com/chrisfromwork/azure-spatial-anchors-samples
```

Comment:
If we're going to take a dependency on this one (which is cool) instead of the official one I'm curious if there's anywhere that we can leave a comment/explain why this is necessary.

Doesn't seem like gitmodules has a comment capability though, so maybe this isn't doable that easily.

Reply:
We're going to do this for now to unblock the feature branch. I am working with the ASA team to get the changes we need into their codebase in parallel to this work.